randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -681,7 +697,10 @@ Extra - map_name: rm_a2a11A
   * Layers: default
   * Open Passage to Inner Alpha Nest South/Dock to Wave Beam Chamber Access Water Pipe
   > Dock to Autrack Corridor Left Water Pipe
-      Hi-Jump Boots or Space Jump or Walljump (Intermediate) or Walljump (Advanced) or Disabled Entrance Rando or Can Use Spider Ball
+      Any of the following:
+          Space Jump or Walljump (Advanced) or Can Use Spider Ball
+          Morph Ball and Disabled Entrance Rando
+          Hi-Jump Boots and Walljump (Intermediate)
```

Comment:
The diff here looks a bit weird, this is this connection/room, right?
![grafik](https://github.com/randovania/randovania/assets/38186597/a6d75ea8-0a47-4006-8963-fac55e97b65a)

Reply:
it probably looks weird because the incorrect `OR` on the `Hijump and WJ (intermediate)`

Reply:
oh, yeah thatd do it.

Reply:
forgot to actually review whoops
add the metroid death check here too please

Reply:
I did, should have `after metroid area 2- interior lower alpha` on all left pipes.
Rooms include inner alpha nest south, wave beam chamber access water pipe, autrack corridor left water pipe, and varia chamber access.

Reply:
github moment, didnt show that it was outdaed for some reason